octobot_trading/personal_data/orders/types/limit/limit_order.py

```diff
@@ -28,6 +28,14 @@ def __init__(self, trader, side=enums.TradeOrderSide.BUY):
         self.trigger_above = self.side is enums.TradeOrderSide.SELL
         self.allow_instant_fill = True
 
+    def update_from_raw(self, raw_order):
+        updated = super().update_from_raw(raw_order)
+        if enums.ExchangeConstantsOrderColumns.TRIGGER_ABOVE.value in raw_order \
+                and self.trigger_above != raw_order[enums.ExchangeConstantsOrderColumns.TRIGGER_ABOVE.value]:
+            updated = True
+            self.trigger_above = raw_order[enums.ExchangeConstantsOrderColumns.TRIGGER_ABOVE.value]
+        return updated
```

Comment:
Do we have an exchange that returns it? I thought it was an internal attribute

Reply:
We need it for conditional orders such as on bybit when we create a stop loss and a take profit, both are counted as stop orders by bybit but one is triggered above the price, one is triggered bellow (defined by the triggerDirection value in https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_placeorder)

Reply:
Ok :+1: Do the Bybit tentacle set a boolean value in ExchangeConstantsOrderColumns.TRIGGER_ABOVE?

Reply:
yes https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/749/files#diff-b1f7ca51f802dd511207722b37f4f1cdde43287523690b6049c608e94f93457dR161